ctapipe/io/hdf5tableio.py

```diff
@@ -127,14 +127,14 @@ def __init__(
         self._group = "/" + group_name
         self.filters = filters
 
-        self.log.debug("h5file: %s", self._h5file)
+        self.log.debug("h5file: %s", self.h5file)
 
     def open(self, filename, **kwargs):
         self.log.debug("kwargs for tables.open_file: %s", kwargs)
-        self._h5file = tables.open_file(filename, **kwargs)
+        self.h5file = tables.open_file(filename, **kwargs)
```

Comment:
Any reason in particular to expose the h5file?

Reply:
yes, it was already used publicly in a number of places (causing lots of pylint warnings), so for now the simplest thing is to make it public as it's clearly useful. There isn't really a good reason to make it private, unless we later add some new API functions to replace use these use cases.